taxi/admin.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Manufacturer, Car, Driver
+
+
+@admin.register(Manufacturer)
+class ManufacturerAdmin(admin.ModelAdmin):
+    search_fields = ["country"]
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    search_fields = ["model"]
+    list_filter = ["manufacturer"]
+
+
+@admin.register(Driver)
+class DriverAdmin(UserAdmin):
+    list_display = UserAdmin.list_display + ("licence_number",)
+    fieldsets = UserAdmin.fieldsets + (
+        (
+            "Licence data",
+            {"fields": ("licence_number",)}
+        ),
+    )
+    add_fieldsets = (
+        (
+            None,
+            {
+                "classes": ("wide",),
+                "fields": ("username", "password1", "password2", "licence_number"),
```

Comment:
Please tell me how to be. Override "add_fieldsets" or add new values to it? Purely logically, you need to redefine so that all columns with data are in 1 place. Please give an answer

Reply:
It is OK to do as you did.